lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -344,7 +344,7 @@ SPIRVEntry *LLVMToSPIRVDbgTran::transDbgEntryImpl(const MDNode *MDN) {
 
     // Compilation unit
     case dwarf::DW_TAG_compile_unit:
-      return transDbgCompilationUnit(cast<DICompileUnit>(DIEntry));
+      return transDbgCompileUnit(cast<DICompileUnit>(DIEntry));
```

Comment:
I see this name is DebugCompilationUnit in the Spec. Can you please check? Thanks

Reply:
Yes, I used the logic Dmitry described. So no more changes are expected here

Reply:
It might be ideal to have the same name in both. But I think that is for a different PR. Thanks